stonesoup/deleter/multi.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Contains the multi-deleter feeders"""
+
+from ..base import Property
+from .base import Deleter
+
+
+class MultiDeleterIntersect(Deleter):
+    """ Track deleter composed of multiple deleters.
+
+    Deletes tracks if they satisfy the conditions of all deleters applied.
```

Comment:
I suspect that logically there should be an option for the intersection or union of all deleter's to make more generic. Could be a logical input along lines of default intersection = True

Reply:
@rjgreen-dstl should read to the end of the file before writing comments

Reply:
I suppose there could be a second argument to a more generic "CompositeDeleter" class which determines whether it intersects or unifies?